### PR TITLE
Add thumbnail url

### PR DESCRIPTION
**Pull request by 0xWumbo**

### CODE REVIEW

File: src/app.controller.ts [hunk 1]
```diff
@@ -5,13 +5,25 @@ import { AppService } from './app.service';
...
+  @Get('v1/check/:tokenId')
+  findOneAzuki(@Param('tokenId') tokenId: string) {
+    return this.appService.getAzukiDetails(parseInt(tokenId));
+  }
+
+  // TODO: Deprecate
   @Get('unclaimed')
   getUnclaimed() {
     return this.appService.getUnclaimed();
   }
```
Comment (0xWumbo) [inline src/app.controller.ts:23]: Keeping these old endpoints for now for backward compatibility. Will remove in later PR

Comment (0xFonzy) [reply src/app.controller.ts:23]: Versioning is perfect thank you!

File: prisma/schema.prisma [hunk 2]
```diff
@@ -3,16 +3,18 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+  provider          = "postgresql"
+  url               = env("DATABASE_URL")
+  shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
```
Comment (0xFonzy) [inline prisma/schema.prisma:8]: we need to add this new env variable to vercel

**Review by 0xFonzy — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 2 above)
Comment (0xFonzy) [reply prisma/schema.prisma:8]: added to vercel

**Review by 0xFonzy — APPROVED** [review]

**Pull request merged**